src/Page.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Input from './Input';
+import TodoList from './TodoList';
```

Comment:
```suggestion
import React from 'react';
import PropTypes from 'prop-types';

import Input from './Input';
import TodoList from './TodoList';
```

이런식으로 의미상으로 분리될 때는 빈줄을 넣어서 구분해주시면 코드 가독성이 더 좋아질 것 같아요;)

Reply:
넴 반영하겠습니다 ㅎㅎ 감사합니다